cni/network/network.go

```diff
@@ -186,6 +190,23 @@ func (plugin *netPlugin) getPodInfo(args string) (string, string, error) {
 	return k8sPodName, k8sNamespace, nil
 }
 
+func setCustomDimensions(cniMetric *telemetry.CNIReport, nwCfg *cni.NetworkConfig, err error) {
```

Comment:
not sure if you need to check if the other pointers are nil here

Reply:
no worries..not needed..nothing can be nil..